content/_changelogs/11.1.0.md

```diff
@@ -0,0 +1,30 @@
+## 11.1.0
+
+_Released 11/14/2022_
+
+**Features:**
+
+- Next.js v13 is now supported.
+  [#24396](https://github.com/cypress-io/cypress/pull/24396)
+
+**Bugfixes:**
+
+- Fixed an 11.0.0 regression where Typescript projects couldn't migrate to 11 without going
+  through 10 first. Fixes
+  [#24643](https://github.com/cypress-io/cypress/issues/24643)
+- Fixed an 11.0.0 regression where `cypress run` crashes when using the junit
+  reporter and an assertion fails. Fixes
+  [#24652](https://github.com/cypress-io/cypress/issues/24652)
+- Fixed Typescript types for `testIsolation`. Fixes
+  [#23911](https://github.com/cypress-io/cypress/issues/23911)
+- Sessions will no longer be stored if validation fails. Fixes
+  [#24208](https://github.com/cypress-io/cypress/issues/24208)
+- Updates were made to enhance the cy.session() recovered error experience. With
+  this change, cy.session() will no longer fail validation when false is
+  returned from the validate callback. Fixes
+  [#21398](https://github.com/cypress-io/cypress/issues/21398),
+  [#24205](https://github.com/cypress-io/cypress/issues/24205), and
+  [#24593](https://github.com/cypress-io/cypress/issues/24593).
```

Comment:
We had chatted about this on slack. This is more of an enhancement vs bug fix, but this should prob be called out under `Experimental Breaking Changes:` with the change to return false